common/src/test/java/com/microsoft/identity/common/MicrosoftStsAccountCredentialAdapterTest.java

```diff
@@ -71,8 +71,6 @@
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.Mockito.when;
 
-@PowerMockIgnore("javax.crypto.*")
-@RunWith(PowerMockRunner.class)
```

Comment:
What's this change about?

Reply:
PowerMockRunner is not used in this class apart from the annotation at the top of the class. This was causing a crash that got fixed when I removed this line.